lib/library_web/views/page_view.ex

```diff
@@ -1,3 +1,23 @@
 defmodule LibraryWeb.PageView do
   use LibraryWeb, :view
+
+  def results_text(assigns) do
+    case Map.get(assigns, :web) do
+      false ->
+        "In the dwyl library"
+      "web" ->
+        "Results from the web"
+      _ ->
+        "No results found, showing results from the web"
+    end
+  end
+
+  def search_the_web_url(conn) do
+    web = Map.get(conn.query_params, :web) || "&web=true"
```

Comment:
Oops there's a mistake here. The left side of the || isn't a string. Will fix.

Reply:
Nice catch, will wait to merge.